tools/distrib/python/grpcio_tools/grpc/__init__.py

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env python2.7
```

Comment:
Everything under this directory is library code and should not have a shebang. I can't speak for the rest of the tool files, but at least some others beyond this directory are lib files rather than executable scripts.